src/main/java/com/google/devtools/build/lib/analysis/actions/SpawnAction.java

```diff
@@ -112,8 +110,8 @@ public class SpawnAction extends AbstractAction implements CommandAction {
 
   private final ResourceSetOrBuilder resourceSetOrBuilder;
   private final ImmutableMap<String, String> executionInfo;
-  private final Consumer<Pair<ActionExecutionContext, List<SpawnResult>>> resultConsumer;
-  private final boolean stripOutputPaths;
+  private final ResultConsumer resultConsumer;
```

Comment:
What is this refactoring for?

Reply:
This will go away with the resolution of merge conflicts as @justinhorvitz has gone rid of it.